src/firmwareanalysis/README.md

```diff
@@ -0,0 +1,5 @@
+# Azure CLI Firmwareanalysis Extension #
+This is an extension to Azure CLI to manage Firmwareanalysis resources.
+
+## How to use ##
+Please add commands usage here.
```

Comment:
please complete the basic usage scenarios to the users

Reply:
Added all commands in readme